src/Invoice.php

```diff
@@ -59,7 +59,7 @@ public static function createReqParams()
      * 'id'=> string,
      * 'user_id'=> string,
      * 'external_id'=> string,
-     * 'status'=> string,
+     * 'status'=> 'EXPIRED',
```

Comment:
Only one status?

Reply:
Yes, according to the API Reference

Reply:
Please ask the Invoice team if they have more status. Because it is weird if they only have one status. Reach them in #invoice-dev

Reply:
I think because it's an expire invoice function, should it be other than 'EXPIRED'? Other statuses are `'PENDING'`, `'PAID'`, `'SETTLED'`, `'EXPIRED'`

Reply:
Ah i see. You're right. Resolving this one